### PR TITLE
Add .gitignore and remove ksetaCPack dependency for cmake 3.9.2

### PR DESCRIPTION
**Pull request by SchellDa**
Adding .gitignore and remove ksetaCPack include (not working with cmake 3.9.2)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+build/
```
Comment (mharrend) [inline .gitignore]: You should not have build in your src tree. ;-)
But we should state this in the documentation.

File: CMakeLists.txt [hunk 2]
```diff
@@ -50,7 +50,7 @@ option(kseta_documentation "Whether or not to create doxygen doc target.")
 include(CTest)
 
 #--- enable CPack --------------------------------------------------------------
-include(cmake/ksetaCPack.cmake)
+#include(cmake/ksetaCPack.cmake)
```
Comment (mharrend) [inline CMakeLists.txt:53]: Do we really would like to remove it for all for now?
Maybe better add an issue first...

File: CMakeLists.txt [hunk 3]
```diff
@@ -50,7 +50,7 @@ option(kseta_documentation "Whether or not to create doxygen doc target.")
 include(CTest)
```
Comment (mharrend) [inline CMakeLists.txt]: te

**Review by mharrend — CHANGES_REQUESTED** [review]
Test again. Please change PR according to comments

File: CMakeLists.txt (hunk 2 above)
Comment (mharrend) [reply CMakeLists.txt:53]: Opened issue #4 to keep track of this.

**Review by hegner — CHANGES_REQUESTED** [review]
please do the changes as suggested by @mharrend

**Review by hegner — APPROVED** [review]
thanks

**Pull request merged**